scss/_reboot.scss

```diff
@@ -268,6 +268,11 @@ a {
   }
 }
 
+// dropdown-item focus
+a.dropdown-item:focus {
```

Comment:
This shouldn't be in `_reboot.scss` since it's using a component selector.

Reply:
Could you please explain to me why ``_reboot.scss`` is the wrong file? I'm a new bootstrap contributor, so I'm asking. 😅